tests/sentry/tempest/test_tempest.py

```diff
@@ -0,0 +1,113 @@
+from unittest.mock import patch
+
+from sentry.tempest.tasks import fetch_latest_item_id, poll_tempest_crashes
+from sentry.testutils.cases import TestCase
+from sentry.utils import json
+
+
+class TempestTasksTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project()
+        self.credentials = self.create_tempest_credentials(self.project)
+
+    @patch("sentry.tempest.tasks.fetch_latest_id_from_tempest")
+    def test_fetch_latest_item_id_task(self, mock_fetch):
+        mock_fetch.return_value = "20001"
+        fetch_latest_item_id(self.credentials.id)
+
+        self.credentials.refresh_from_db()
+        assert (
+            self.credentials.latest_fetched_item_id == "20001"
+        )  # Since the ID is stored as a string
+        mock_fetch.assert_called_once()
+
+    @patch("sentry.tempest.tasks.fetch_latest_id_from_tempest")
+    def test_fetch_latest_item_id_error(self, mock_fetch):
+        mock_fetch.return_value = "Invalid credentials"
```

Comment:
I don't know how response from tempest look like, but maybe it's better if we always return JSON objects, with key "error" if something bad happened:
```
{
  "error": "Invalid credentials"
}
```

That way we can always expect JSON response and just read the keys we need